docs/src/roadmap.md

```diff
@@ -11,14 +11,14 @@ JuMP 1.0 will be ready to release roughly when all of these tasks are completed.
 Some but not all of these tasks are summarized in the
 [JuMP 1.0 milestone](https://github.com/jump-dev/JuMP.jl/milestone/12).
 
-- Create a website for JuMP.
+- Create a website for JuMP. (**Done**: https://jump.dev)
 - Deprecate the JuliaOpt organization and move repositories to the
-  [JuMP-dev](https://github.com/JuMP-dev) organization.
+  [JuMP-dev](https://github.com/JuMP-dev) organization. (**Done**)
 - Address major regressions from JuMP 0.18.
   - Performance ([#1403](https://github.com/jump-dev/JuMP.jl/issues/1403),
                  [#1654](https://github.com/jump-dev/JuMP.jl/issues/1654),
                  [#1607](https://github.com/jump-dev/JuMP.jl/issues/1607))
-  - Callbacks
+  - Callbacks (**Done**: see `examples/callbacks.jl`)
```

Comment:
I think done for v1. There is a way forward to add more functionality in future point releases.